services/shotgrid_common/utils.py

```diff
@@ -14,11 +15,50 @@
 from ayon_api.entity_hub import ProjectEntity
 from ayon_api.utils import slugify_string
 from ayon_api import get_attributes_for_type
+import logging
 
-from nxtools import logging
 import shotgun_api3
 
 
+def get_logger(name: str) -> logging.Logger:
+    """Return a logger instance with the given name."""
+    # get environment variable DEBUG level
+    log_level = os.environ.get("LOGLEVEL", "INFO").upper()
+
+    logger = logging.Logger(name)
+    # create console handler and set level to debug
+    ch = logging.StreamHandler()
+    ch.setLevel(log_level)
+
+    formatting_str = (
+        "%(asctime)s.%(msecs)03d %(levelname)s: %(message)s"
+    )
+
+    if log_level == "DEBUG":
+        formatting_str = (
+            "%(asctime)s.%(msecs)03d | %(module)s | %(funcName)s | "
+            "%(levelname)s: %(message)s"
+        )
+
+    # create formatter
+    formatter = logging.Formatter(
+        fmt=formatting_str,
+        datefmt="%Y-%m-%d %H:%M:%S",
+    )
+
+    # add formatter to ch
+    ch.setFormatter(formatter)
+
+    # add ch to logger
+    logger.addHandler(ch)
+
+    return logger
+
+
+# create logger
+log = get_logger(__name__)
```

Comment:
I am not so sure about correct place for this function and also if this is the best way to create the logger @iLLiCiTiT or @martastain you might have perhaps better idea?